panther_analysis_tool/destination.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class FakeDestination:
+    """Stub class as a replacement for the Destination class
+    that wraps alert output metadata."""
+
+    destination_id: str
+    destination_display_name: str
```

Comment:
Perhaps we could make some changes to the BE `outputs-api` to allow anyone to get a list of the destination display names, that way we could at least check to see if a destination display name matches some entry in the API.

Thoughts? @nhakmiller  @lindsey-w

Reply:
I assume that destinations rarely change, right? Perhaps only an export of the id/name in YAML could suffice.

Reply:
Getting or exporting a list would both work. My concern with having `pat` retrieve the list is we'd need to update the IAM permissions of the `pat` role for all customers. The public API can't come fast enough! For now the solution of allowing users to specify destinations with a CLI flag seems good.